ReText/window.py

```diff
@@ -1239,6 +1247,21 @@ def viewHtml(self):
 		htmlDlg.raise_()
 		htmlDlg.activateWindow()
 
+	def insertImage(self):
+		supportedExtensions = ['.png', '.jpg', '.jpeg', '.gif', '.bmp']
+		fileFilter = ' (' + str.join(' ', ['*' + ext for ext in supportedExtensions]) + ');;'
+		fileName = QFileDialog.getOpenFileName(self,
+												 self.tr("Select one or several files to open"), QDir.currentPath(),
+												 self.tr("Supported files") + fileFilter + self.tr("All files (*)"))
+
+		fileName = fileName[0]
+		imageText = '![{0}]({1})'.format(self.tr('Alt text'), self.tr(fileName))
```

Comment:
Please also add syntax for reStructuredText and Textile. There is this block in ReText/editor.py:
https://github.com/retext-project/retext/blob/27a2745e3ae65bf894f359c5e05fe0559a65a99d/ReText/editor.py#L416-L422

Probably it should be split out to a separate function/method and reused here.

Reply:
Sure!